packages/xrpl/.eslintrc.js

```diff
@@ -84,7 +84,8 @@ module.exports = {
         'max-statements': 'off',
         // Snippets have logs on console to better understand the working.
         'no-console': 'off',
-        'import/no-extraneous-dependencies': 'off',
+        '@typescript-eslint/no-unsafe-member-access': 'off',
```

Comment:
What were the places was this rule was failing?

Reply:
It might have been an artifact - nothing is failing with those rules in place, so I'll add them back in.